app/classes/document_query.rb

```diff
@@ -65,6 +71,32 @@ def aggregation_hash(facet_field)
     }
   end
 
+  def date_aggregation_hash(date_facet_field)
+    {
+      date_range: {
+        field: date_facet_field,
+        format: '8M/d/u',
+        ranges: [
+          {
+            key: 'Last Week',
+            from: 'now-1w/m',
+            to: 'now/m'
+          },
+          {
+            key: 'Last Month',
+            from: 'now-1M/m',
+            to: 'now-1w/m'
```

Comment:
Should all ranges have `to: 'now/m'`, or is there a reason the `to` date is different for reach range?

Reply:
@ddzz Good question, and I'm drawing in @amanifarooque to weigh in on the product perspective since I realize I may have misinterpreted the JIRA ticket and PRD upon re-review since this isn't explicitly stated in either of those locations.

@amanifarooque As this is currently implemented, the date-based buckets do not - by design - contain any shared results.  But, this may stem from my own (mistaken) assumption about the desired functionality.  So, question:

If you have a document with a date in the last week (say, 2022-11-07), should that document:
1. ONLY appear in the "Last Week" bucket, but not in "Last Month" or "Last Year"; or,
2. Appear in ALL 3 facet buckets since that date is technically from all of the above - "Last Week", "Last Month", and "Last Year".

The code, as stands, makes unique buckets - so option 1 above.

But, after looking over the PRD and JIRA again, option 2 may be what is desired (and what @ddzz is getting at in the question above).

Either way, very minimal change if I'm wrong here.

Reply:
Ah good catch, I was assuming that the "end" date for all of these ranges would be "now" - so there would be overlap in the buckets. I believe this would be in line with expected user behavior for these filters.

Reply:
@ddzz This has been updated to reflect the above.  Thanks so much for catching that.

Reply:
@ddzz Yep, another good catch.  I've updated, as well as slightly tweaked the spec to ensure the search only returns the document we want (previously we got away with reusing the same query because the test documents were setup within a minute of each other).

Reply:
To be consistent I think we should take out the `/m`s in the `from:` dates in the ranges.

Reply:
@ddzz Done!